pkg/operator/controller/ingress/status_test.go

```diff
@@ -877,6 +914,27 @@ func Test_computeLoadBalancerProgressingStatus(t *testing.T) {
 			platformStatus: awsPlatformStatus,
 			expectStatus:   operatorv1.ConditionFalse,
 		},
+		{
+			name:           "LoadBalancerService, OpenStack floating IP spec and status are empty",
+			ic:             loadBalancerIngressControllerWithFloatingIP("", ""),
+			service:        lbServiceLoadBalancerIP(""),
+			platformStatus: openstackPlatformStatus,
+			expectStatus:   operatorv1.ConditionFalse,
+		},
+		{
+			name:           "LoadBalancerService, OpenStack floating IP spec and status are the same",
+			ic:             loadBalancerIngressControllerWithFloatingIP("192.168.0.1", "192.168.0.1"),
+			service:        lbServiceLoadBalancerIP("192.168.0.1"),
+			platformStatus: openstackPlatformStatus,
+			expectStatus:   operatorv1.ConditionFalse,
+		},
+		{
+			name:           "LoadBalancerService, OpenStack floating IP spec and status are not the same",
+			ic:             loadBalancerIngressControllerWithFloatingIP("192.168.0.1", ""),
+			service:        lbServiceLoadBalancerIP("192.168.0.1"),
+			platformStatus: openstackPlatformStatus,
+			expectStatus:   operatorv1.ConditionTrue,
+		},
```

Comment:
Why do you need `lbServiceLoadBalancerIP`?  `computeLoadBalancerProgressingStatus` doesn't look at the service's `spec.loadBalancerIP`.  You can substitute `lbService` in these test cases, and they still pass.